pkgs/applications/networking/cluster/terraform-providers/shell/default.nix

```diff
@@ -1,32 +0,0 @@
-{ lib, fetchFromGitHub, buildGoModule }:
-buildGoModule rec {
-  pname = "terraform-provider-shell";
-  version = "1.6.0";
-
-  src = fetchFromGitHub {
-    owner = "scottwinkler";
-    repo   = pname;
-    rev    = "v${version}";
-    sha256 = "0jxb30vw93ibnwz8nfqapac7p9r2famzvsf2h4nfbmhkm6mpan4l";
-  };
-
-  vendorSha256 = "1p2ja6cw3dl7mx41svri6frjpgb9pxsrl7sq0rk1d3sviw0f88sg";
-
-  doCheck = false;
-
-  subPackages = [ "." ];
-
-  # Terraform allows checking the provider versions, but this breaks
-  # if the versions are not provided via file paths.
-  postInstall = "mv $out/bin/${pname}{,_v${version}}";
```

Comment:
Is this lost?

Reply:
It's not lost. It's done in [`teraform-providers/default.nix:L23`](https://github.com/NixOS/nixpkgs/blob/a9e03c40a1da210c17e3ea6f44a29e443c9d46ab/pkgs/applications/networking/cluster/terraform-providers/default.nix#L23).

The old `shell/default.nix` file replicates what's done in `default.nix`. Btw, the historical reason why `shell/default.nix` is in its own file is that it used `buildGoModule`, which was not supported by [the "standard model"](https://github.com/NixOS/nixpkgs/blob/a9e03c40a1da210c17e3ea6f44a29e443c9d46ab/pkgs/applications/networking/cluster/terraform-providers/default.nix#L43).